api/src/main/java/org/openmrs/util/OpenmrsConstants.java

```diff
@@ -1078,6 +1078,9 @@ public static String getOpenmrsProperty(String property) {
 		
 		props.add(new GlobalProperty("allergy.concept.unknown", "1067AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA",
 		        "UUID for the allergy unknown concept"));
+
+		props.add(new GlobalProperty(LOGIN_OWA_URL, "/openmrs/login.htm",
```

Comment:
The name LOGIN_OWA_URL means that this cannot be handled by anything other than OWAs